### PR TITLE
RA-1478:Adding ability to filter by encounter type and ability to sho…

### PR DESCRIPTION
**Pull request by reagan-meant**
RA-1478:Adding ability to filter by encounter type and ability to show empty encounters in ObsAcrossEncounters Dashboard widget.
https://issues.openmrs.org/browse/RA-1478
Still working on it.....

### CODE REVIEW

**Comment by reagan-meant** [comment]
Unfortunately my ui is breaking when i try to do local testing though the changes are being reflected...the error i get in chrome console on inspection is "Failed to load resource: the server responded with a status of 404 (Not Found) jquery-3.4.1.min.js:1 "

**Comment by reagan-meant** [comment]
The ui problem was fixed in the recent refapp updates...Everything works fine now as intended by this ticket...Screenshot attached to the ticket.

File: omod/src/main/resources/apps/dashboardWidgets_app.json [hunk 1]
```diff
@@ -33,6 +33,8 @@
       "icon": "icon-user-md",
       "label": "HEALTH TREND SUMMARY",
       "concepts": "5087AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,5088AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,5089AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+      "encounterType": "67a71486-1a54-468f-ac3e-7091a9a79584",
```
Comment (dkayiwa) [inline omod/src/main/resources/apps/dashboardWidgets_app.json]: Which encounter type is this?

Comment (reagan-meant) [reply omod/src/main/resources/apps/dashboardWidgets_app.json]: Vitals

**Pull request closed without merging**